src/swing.ts

```diff
@@ -80,9 +80,10 @@ export interface Stack {
    * Creates an instance of Card and associates it with an element.
    *
    * @param {HTMLElement} element
+   * @param {boolean} prepend
    * @return {Card}
    */
-  createCard(elment: HTMLElement): void;
+  createCard(elment: HTMLElement, prepend: boolean): void;
```

Comment:
@mashhoodr I'm not sure about the rules for providing defaults in interfaces - I will test more this evening with no prepend to confirm.